docker/dev/mysql.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   mysql:
-    image: mysql:5.7
+    image: mysql:latest
```

Comment:
I would suggest using a specific version like 8.0.3.
Using latest can cause issues if mysql team bumps latest tag to a new version.

Reply:
Yes you are right. I've merged a new PR with the 8.0 version. This PR should be abandoned.